lib/foo.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function foo() {
+  return 'Hello world!';
+}
+
+module.exports = {
+  foa: foo,
```

Comment:
Actually this is not a typo, eslint was complaining so I changed it to `foa` (originally it was `foo:foo`). Object property shorthand is probably the best way, thanks!